pytorch_pfn_extras/onnx/strip_large_tensor.py

```diff
@@ -1,43 +1,42 @@
-# mypy: ignore-errors
-
 import argparse
 import json
+import onnx
 import operator
 from functools import reduce
-
-import numpy
+from typing import Any
```

Comment:
Now this line needs to be removed https://github.com/pfnet/pytorch-pfn-extras/runs/4503061809?check_suite_focus=true#step:5:67

```suggestion
```